### DISCUSSION TITLE
Remove grey background

### DISCUSSION THREAD

**Discussion by murtukov** [community/community#3952] [Discussions]
While the background of the initial discussion message and the answers is white, the background of "replies to answers" is grey, which reduces readability, especially when there are many markdown code snippets and citations. It would be nice to remove the grey background (IMHO).
![image](https://user-images.githubusercontent.com/19292777/118283659-e483b980-b4cf-11eb-98fd-2108625a2c82.png)

**Comment by borekb** [comment]
Agree, and code blocks suffer from this the most as their gray is exactly the same as the background:

<img width="1021" alt="Screen Shot 2021-12-08 at 11 46 53" src="https://user-images.githubusercontent.com/101152/145195736-f1953103-6da1-477d-9610-a9b1fc449aaf.png">

Source: https://github.com/github/feedback/discussions/8311#discussioncomment-1771776
